packages/tooling/__tests__/lib/flatten-schema.test.js

```diff
@@ -87,3 +88,166 @@ describe('$ref usages', () => {
     expect(flattenSchema(schema, oas)).toStrictEqual(expected);
   });
 });
+
+describe('polymorphism cases', () => {
+  describe('allOf', () => {
```

Comment:
So flattenSchema is for use in the response sidebar? 👌

Reply:
Yep, that's the only place it's used.